test/runtests.jl

```diff
@@ -1478,228 +1572,228 @@ end
         end
     end
 
-    @testset "Statements" begin
-        @testset "No Params, Output" begin
-            conn = LibPQ.Connection("dbname=postgres user=$DATABASE_USER"; throw_error=true)
+    # @testset "Statements" begin
```

Comment:
Mistaken commit of commenting-out? I think this might be the source of some of the coverage changes.